src/humanPose/draw.js

```diff
@@ -643,9 +643,11 @@ function updateJointsHistorical(poseRenderer, poseObject) {
 function updateJoints(poseRenderer, poseObject) {
     let groundPlaneRelativeMatrix = getGroundPlaneRelativeMatrix();
 
-    let objectRootMatrix = new THREE.Matrix4();
-    setMatrixFromArray(objectRootMatrix, poseObject.matrix);
-    groundPlaneRelativeMatrix.multiply(objectRootMatrix);
+    if (poseObject.matrix && poseObject.matrix.length > 0) {
```

Comment:
Should we even render if this matrix is not in valid state yet?

Reply:
On the phone the matrix will never be updated to a valid matrix